src/mongo/mongoConnectionStrings.ts

```diff
@@ -63,23 +63,16 @@ export async function parseMongoConnectionString(connectionString: string): Prom
         }
     }
 
-    const serverConfig: Server | ReplSet | Mongos = mongoClient.db(testDb).serverConfig;
-    // Azure CosmosDB comes back as a ReplSet
-    if (serverConfig instanceof ReplSet) {
```

Comment:
All the mongo attached accounts are not passing this condition as ReplSet is deprecated. 
However, the code to access `host` and `port` values hasn't changed.